### PR TITLE
feat: recipes counter

### PR DESCRIPTION
**Pull request by taylorichigo**

### CODE REVIEW

**Comment by lonba** [comment]
**FYI**
Commit /[22262bb](https://github.com/moncapitaine/cooking-quiz/pull/34/commits/22262bbf924e57d3607b591439ec355e9ed3823b)/ is from mine VS Code user: it seems some configuration for my user lonba is missing! My bad :(

File: components/recipesCounter/allRecipesOutput.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+export const AllRecipesOutput = (props: {value: number}) => {
```
Comment (lonba) [inline components/recipesCounter/allRecipesOutput.tsx:3]: Maybe the component name is a little missleading:

- [ ] does it means it output **all** recipes in a big rendered HTML? 
- [ ] or should this just show how many recipe are available (amount)

Does this match with our Coding Guidelines / Clean Coding? @moncapitaine

Comment (lonba) [reply components/recipesCounter/allRecipesOutput.tsx]: Changes are fine by me

**Review by lonba — APPROVED** [review]

**Comment by lonba** [comment]
Can you please resolve the merge conflicts, berfore i continue with my review?

File: components/recipesCounter/allRecipes.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const allRecipes = [
```
Comment (moncapitaine) [inline components/recipesCounter/allRecipes.ts:1]: rename file to testRecipes.ts

File: components/recipesCounter/countAllRecipesOutput.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+export const CountAllRecipesOutput = (props: {value: number}) => {
```
Comment (moncapitaine) [inline components/recipesCounter/countAllRecipesOutput.tsx:3]:
```suggestion
export const RecipesCounter = (props: {value: number}) => {
```

plus rename file to recipesCounter.tsx

**Review by lonba — APPROVED** [review]

File: components/recipesCounter/recipesCounter.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+export const RecipesCounter = (props: {value: number}) => {
+    return(
+        <div className="p-6 bg-white border-4 border-solid border-blue-400">
+            <p>Found {props.value} Recipes</p>
```
Comment (moncapitaine) [inline components/recipesCounter/recipesCounter.tsx:6]: if only 1 recipe then please render "Found 1 recipe"

**Review by moncapitaine — APPROVED** [review]

**Comment by taylorichigo** [comment]
closes #27

**Review by moncapitaine — APPROVED** [review]

**Pull request merged**